sdk/src/main/java/co/omise/android/SDKLog.kt

```diff
@@ -0,0 +1,25 @@
+package co.omise.android
+
+import android.util.Log
+
+internal object SDKLog {
+    private const val TAG = "co.omise.android"
+
+    fun d(message: String) {
+        if (BuildConfig.DEBUG) {
+            Log.d(TAG, message)
+        }
+    }
+
+    fun e(message: String, e: Throwable) {
+        if (BuildConfig.DEBUG) {
+            Log.e(TAG, message, e)
+        }
+    }
+
+    fun wtf(message: String, e: Throwable) {
```

Comment:
What did you mean here? What the 🦊 😄

Reply:
**What a Terrible Failure** 😂 
https://developer.android.com/reference/android/util/Log#wtf(java.lang.String,%20java.lang.Throwable)

Reply:
:D